src/format-importer.ts

```diff
@@ -147,7 +147,9 @@ export abstract class FormatImporter {
 	 * Recursively create folders, if they don't exist.
 	 */
 	async createFolders(path: string): Promise<TFolder> {
-		let normalizedPath = normalizePath(path);
+		// can't create folders starting with a dot
+		const sanitizedPath = path.replace(/^\./g, '').replace(/\/\./g, '/');
```

Comment:
```suggestion
		const sanitizedPath = path.split('/').map(n => n.replace(/^\.+/, '')).join('/');
```
Suggest changing to a split/join and processing each part separately, and also adding `+` to the regex in case there were multiple dots.